lib/config.py

```diff
@@ -96,6 +97,10 @@ def _parse_config(self, config: JSON) -> None:
             # Convert hours to seconds
             self.retrieval_interval *= 3600
 
+        if "healthchecks_url" in config:
+            self.healthchecks_url = config["healthchecks_url"]
+            logger.debug("Setting healthchecks URL to %s", self.healthchecks_url)
```

Comment:
This should be checked to make sure it is a string and probably not logged as it is private information (I want to avoid logging private info to make log sharing as easy as possible) and it isn't necessary information. Instead a message saying something like `A Healthchecks URL has been provided` would be sufficient.

Here's how the type checking can be done (replacing `browser_path` with `healthchecks_url`:
```py
 if not isinstance(self.browser_path, str):
                raise ConfigError("'browser_path' must be a string")
```